app/src/main/res/layout/top_site_item.xml

```diff
@@ -2,76 +2,64 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<FrameLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/top_site_item"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:focusable="true">
```

Comment:
I think we might want to keep the `focusable` here, which was added in https://github.com/mozilla-mobile/fenix/issues/27004

Reply:
Should this be in the updated commit or revert commit?

Reply:
Since it was removed in the revert commit, I would expect the change to be undone there so, it doesn't appear like this was magically re-added.